DOCS/faq.md

```diff
@@ -36,127 +12,108 @@ standardized short identifier, the full name, the license text, and a
-* an [explanation of the fields](https://github.com/spdx/license-list-XML/blob/main/DOCS/license-fields.md) used in the SPDX License List
-
-The authoritative files for the SPDX License List are stored in XML source files in a [Github repo](https://github.com/spdx/license-list-XML). These files are used to generated the supported SPDX License List formats, including the web pages and other data formats found in [license-list-data repo](https://github.com/spdx/license-list-data).
-
-##### <a href="#top"> Back to Top</a>
-
-<br>
-
-### <span id="why"></span>Why does it exist?
-
+* [license inclusion principles](https://github.com/spdx/license-list-XML/blob/main/DOCS/license-inclusion-principles.md) 
+regarding what is eligible for inclusion on the SPDX License List
+* [matching guidelines](https://spdx.github.io/spdx-spec/v2.3/license-matching-guidelines-and-templates/) 
+provide parameters as to what constitutes a match to a license or exception on the SPDX License List
+* an [explanation of the fields](https://github.com/spdx/license-list-XML/blob/main/DOCS/license-fields.md) 
+used in the SPDX License List
+* [license expression syntax](LINK) 
+enable expressing composite licensing scenarios, such as when more than one license applies, there is a choice of license, or an exception or additional terms apply to the license
+
+The authoritative files for the SPDX License List are stored in XML source files 
+in a [Github repo](https://github.com/spdx/license-list-XML). These files are 
+used to generated the supported SPDX License List formats, including the 
+web pages and other data formats found in [license-list-data repo](https://github.com/spdx/license-list-data).
+
+There are also certain sections in the SPDX specification related to license information. 
+It is helpful to understand the fields and purpose of these sections, even if you 
+are not generating or consuming an SPDX document. 
+
+## Why does it exist?
 The purpose of the SPDX License List is to enable efficient and reliable
-identification of such licenses and exceptions in an SPDX document, in
-source files or elsewhere. Being able to refer to licenses via the short
+identification of licenses and exceptions in an SPDX document, in
+source files, or elsewhere. In an SPDX document, being able to refer to licenses via the short
 form identifier lessens the SPDX file size and allows for unambiguous
 license identification.
 
 SPDX short identifiers have also proven to be useful in a similar way
-outside of an SPDX document. For example, many open source projects have
-begun to use the short identifier as an accurate, concise, and
+outside of an SPDX document. For example, many open source projects use the short identifier 
+in source files by putting the tag ".... at the top of each file. This provides an accurate, concise, and
```

Comment:
I'm not sure what the `"....` was meant to be -- is this supposed to be something like `SPDX-License-Identifier: ...` ?